test/mpi/f90/timer/wtimef90.f90

```diff
@@ -28,4 +28,5 @@ program main
                 print *, ' No Errors'
           endif
           
```

Comment:
Should above set `err` and let `mtest_finalize` print `No Errors`?

Reply:
I left it as-is because of the more useful error message, but I really don't feel strongly either way. I can make the change.